app/uk/gov/hmrc/incorporatedentityidentificationfrontend/services/JourneyService.scala

```diff
@@ -41,4 +41,12 @@ class JourneyService @Inject()(journeyConnector: JourneyConnector,
       case _ =>
         throw new NotFoundException(s"Journey config was not found for journey ID $journeyId")
     }
+
+  def getCallingService(journeyId: String, authInternalId: String): Future[Option[String]] =
+    journeyConfigRepository.findJourneyConfig(journeyId, authInternalId).map {
+      case Some(journeyConfig) =>
+        journeyConfig.pageConfig.optServiceName
+      case _ =>
+        throw new NotFoundException(s"Journey config was not found for journey ID $journeyId when trying to get calling service")
+    }
```

Comment:
Do we need a separate method here or could we just use the journey config model?

Reply:
I though the same thing, rewrote it and forgot to remove this method!!  I'll get it removed.

Reply:
I've removed this method